tests/common/src/main/resources/northwind/employees.csv

```diff
@@ -0,0 +1,10 @@
+EmployeeID,LastName,FirstName,Title,TitleOfCourtesy,BirthDate,HireDate,Address,City,Region,PostalCode,Country,HomePhone,Extension,Photo,Notes,ReportsTo,PhotoPath
```

Comment:
Yes, NorthWind Trader's schema is widely used in public domain, especially with the database research students. Also this data is shipped with Microsoft products.